dataplane/src/main.rs

```diff
@@ -0,0 +1,3 @@
+fn main() {
+    // placeholder
+}
```

Comment:
Even if it's an empty function, can we have a copyright header + SPDX tag on the file, please?

Reply:
will do

Reply:
Resolved